ingest/genomes/genome_annotation_metadata.py

```diff
@@ -128,12 +124,12 @@ def update_meta_row(row, org_metadata, annot_metadata):
     return new_row
 
 
-def record_annotation_metadata(ensembl_metadata, scp_species):
+def record_annotation_metadata(output_dir, ensembl_metadata, scp_species):
     """Write annotation URLs, etc. to species metadata reference TSV file
     """
     new_metadata_ref = []
 
-    ref_file = 'species_metadata_reference.tsv'
+    ref_file = f'{output_dir}species_metadata_reference.tsv'
```

Comment:
is this a place where os.path.join might be more robust? 
 https://docs.python.org/3/library/os.path.html

Reply:
Yup!  Done.  It's more verbose, but trading that for robustness makes sense.